### PR TITLE
Add messageId to richObject definition of Talk

### PR DESCRIPTION
**Pull request by SystemKeeper**
## Summary

See https://github.com/nextcloud/spreed/pull/9462

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

File: lib/public/RichObjectStrings/Definitions.php [hunk 1]
```diff
@@ -199,6 +199,12 @@ class Definitions {
 					'description' => 'The icon url to use as avatar',
 					'example' => 'https://localhost/ocs/v2.php/apps/spreed/api/v1/room/R4nd0mToken/avatar'
 				],
+				'message-id' => [
+					'since' => '28.0.0',
```
Comment (nickvergessen) [inline lib/public/RichObjectStrings/Definitions.php:203]: Maybe we can do 27.0.0 still via beta2?

Cc @AndyScherzinger

**Review by vitormattos — APPROVED** [review]

**Review by AndyScherzinger — APPROVED** [review]

**Comment by AndyScherzinger** [comment]
I suspect this needs a documentation update, right @nickvergessen ?

**Comment by nickvergessen** [comment]
Currently all of this is not documented elsewhere.
I guess we should try to make it exportable as rst and sync it over like the sample.config.php

**Comment by SystemKeeper** [comment]
So we need to backport to stable27, @nickvergessen ?

File: lib/public/RichObjectStrings/Definitions.php (hunk 1 above)
Comment (tobiasKaminsky) [reply lib/public/RichObjectStrings/Definitions.php]: a bit late to the party, but why was I mentioned?

**Comment by DaphneMuller** [comment]
Hello!
Thank you for your work on this! I noticed that your PR still has the label 'documentation'. I'm not sure if you already had a look at the documentation for your PR, but at Nextcloud we strive to document changes that affect other app developers or other admins before the release takes place, and I'm pinging you with the hope to get this done. 

Are you familiar with our documentation process already? 
Changes that affect app developers should be documented here:
https://docs.nextcloud.com/server/27/developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_27.html

Changes that affect administrators should be documented here:
https://docs.nextcloud.com/server/latest/admin_manual/release_notes/upgrade_to_27.html

While I understand sometimes it's handy to merge fast to get your changes in, it would be great if next time you could try to add the documentation before merging. You can find more information on that here:
https://docs.nextcloud.com/server/27/developer_manual/prologue/compatibility_app_ecosystem.html#documentation-procedures-of-changes-that-affect-app-developers

If all your documentation efforts are done, please remove the label 'documentation' and check the checkbox in your opening note and I'll stop bugging you :) 

Many thanks in advance and thanks again for your work!

**Pull request merged**